app/src/main/java/com/mobileproto/dabrahamsmruehle/scavengerhunt/HUDFragment.java

```diff
@@ -157,7 +174,7 @@ public void onLowMemory()
     public interface OnFragmentInteractionListener
     {
         // TODO: Update argument type and name
-        public void onFragmentInteraction(Object o);
+        public void onFragmentInteraction(Uri u);
```

Comment:
This is going to possibly cause issues when I try to merge my camera code, since I use `onFragmentInteraction` to switch to the camera, but whatever. You wrote your code first, I'll figure out how we'll deal with this when I'm merging. :)